conf.yaml

```diff
@@ -195,7 +195,7 @@ contents:
             prefix:     en/elastic-stack-deploy
             current:    7.10
```

Comment:
Does this need to be updated as well?

Reply:
Thanks, Brandon!  The 7.11 branch for Azure Marketplace doesn't exist.  When it's there, I'll update `current`.  In the meantime, I need to pull 7.10 out of live branches. It's making the docs build go BOOM. :-)

Reply:
💣